index.html

```diff
@@ -3355,14 +3355,19 @@ <h3>Verify Proof Sets and Chains</h3>
         <p>
 Required input is a
 [=secured data document=] (|securedDocument|). A list of
-verification results corresponding to each proof in |allProofs| is
-produced as output.
+[=verification result=]s corresponding to each proof in |allProofs| is
+generated, and a single combined [=verification result=] is returned as output.
+Implementations MAY return any of the other [=verification result=]s or Any
```

Comment:
```suggestion
Implementations MAY return any of the other [=verification result=]s or any
```

Reply:
including @dlongley's fix
```suggestion
Implementations MAY return any of the other [=verification result=]s and/or any
```